meta/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+galaxy_info:
+  author: Michel Weitbrecht
+  description: Installs smartmontools and configures smartd
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+        - yakkety
+        - zesty
+        - artful
+    - name: ArchLinux
+      versions:
+        - all
+        - any
+    - name: Debian
```

Comment:
Is this even tested against Debian?

Reply:
I tested this on all my debian machines.

Reply:
You don't own any debian machines

Reply:
Did you just assume my operating system?

Reply:
Yes. Just fix this issue, it's not that complicated